Dockerfile

```diff
@@ -44,6 +44,10 @@ RUN cd /app/src/autograph && go install .
 RUN cd /app/src/autograph/tools/autograph-monitor && go build -o /go/bin/autograph-monitor .
 RUN cd /app/src/autograph/tools/autograph-client && go build -o /go/bin/autograph-client .
 
+# Extract and verify the Google KMS library.
+RUN cd /tmp && curl -L https://github.com/GoogleCloudPlatform/kms-integrations/releases/download/pkcs11-v1.6/libkmsp11-1.6-linux-amd64.tar.gz | tar -zx --strip-components=1
```

Comment:
Could the version of libkmsp11 be a parameter like the `GO_VERSION`?

Reply:
Yeah, that's a good call. Let's do that.